docs/reference/generated/kubeadm.md

```diff
@@ -67,7 +67,7 @@ following steps:
    See [Securing your installation](#securing-more) for hardening.
 
 1. kubeadm installs add-on components via the API server.  Right now this is
-   the internal DNS server and the kube-proxy DaemonSet.
+   the internal DNS server and the kube-proxy DaemonSet. There is no requirement for specifying kube-proxy bind address in kubeadm configuration file. The kube-proxy bind address is dynamically assigned based on the address type of API server (i.e. v4 will assign 0.0.0.0 & v6 will assign :: ).
```

Comment:
I don't know really if this needs to be said; the user shouldn't have to care about this, right?